packages/schematics/integration/nixt-facade/run-result.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Result of nixt#run.
+ */
+export interface RunResult {
+  readonly code?: number;
+  readonly cmd?: string;
+  readonly err?: number;
+  readonly stdout?: string;
```

Comment:
I think that I miss something about `nixt`.
It seems that `run` also provides `stdout/stderr`.

When do we need to use `execute`?
When do we need to use `run`?

Reply:
see comment https://github.com/co-IT/schematics/pull/35#discussion_r280689361